front/lib/models/assistant/agent.ts

```diff
@@ -185,6 +192,16 @@ AgentConfiguration.belongsTo(AgentRetrievalConfiguration, {
   foreignKey: { name: "retrievalConfigurationId", allowNull: true }, // null = no retrieval action set for this Agent
 });
 
+// Agent config <> DustAppRun config
```

Comment:
Do we want to enforce that Dust app is mutually exclusive with retrieval config ?

Reply:
This is mechanically enforced by our code, but we could use a hook indeed